build(client): upgrade mocha 10 -> 11

tylerbutler opened the pull request:
Upgrade mocha to v11.

No changes were needed because a previous change updated the root package.json to pass a setting to mocha via an env variable, so that change is not needed in this PR.

alexvy86 reviewed (approved):
Looks good to me. If you'll be around next week, probably ok to merge, otherwise I'd suggest we wait until more of us are around to troubleshoot if necessary.

tylerbutler committed:
build(client): upgrade mocha 10 -> 11

Bump mocha from ^10.8.2 to ^11.7.5 across all client release group packages.
This includes two previously missed packages (inventory-app and text-editor).

tylerbutler committed:
build(client): regenerate lockfile

Regenerate pnpm-lock.yaml after mocha upgrade.

tylerbutler committed:
Merge branch 'main' into mocha-upgrade
